distributions/standalone/assembly-whitelist.txt

```diff
@@ -189,6 +189,7 @@
 /mule-standalone-${productVersion}/lib/opt/abdera-parser-1.1.3.jar
 /mule-standalone-${productVersion}/lib/opt/abdera-server-1.1.3.jar
 /mule-standalone-${productVersion}/lib/opt/abdera-spring-1.1.3.jar
+/mule-standalone-${productVersion}/lib/opt/animal-sniffer-annotations-1.14.jar
```

Comment:
what about these extra libs?

Reply:
Those are transitive dependencies in the new versions of Guava